cantools/autosar/secoc.py

```diff
@@ -15,6 +15,36 @@ class SecOCError(Error):
     """
     pass
 
+def compute_authenticator(raw_payload: Union[bytes, bytearray],
```

Comment:
You can just use `bytes`. The [docs](https://docs.python.org/3/library/typing.html#typing.ByteString) say:

> As a shorthand for this type, [bytes](https://docs.python.org/3/library/stdtypes.html#bytes) can be used to annotate arguments of any of the types mentioned above.

Reply:
ah, cool I didn't know that...